ui/lib/core/addon/templates/components/replication-secondary-card.hbs

```diff
@@ -45,34 +45,31 @@
     </h2>
     <h2 class="title is-3 grid-item-right-bottom" data-test-connection>{{connection}}</h2>
   {{else}}
-    <h3 class="title is-5 grid-item-title card-title">{{title}}
-      <ToolTip @verticalPosition="above" @closeDelay="2000ms" as |T|>
-        <T.trigger>
-          <Icon @glyph={{"check-circle-outline" }} @size="m" class={{"has-text-grey"}} />
-        </T.trigger>
-        <T.content @class="tool-tip">
-          <div class="box">
-            There may be a difference between last remote WAL and last WAL entry from primary. If it persists over time,
-            investigate.
-            <LearnLink @path="/vault/operations/monitor-replication#are-my-dr-clusters-in-sync" @class="has-text-white">
-              Learn more
-            </LearnLink>
-          </div>
-        </T.content>
-      </ToolTip>
-    </h3>
+    <h3 class="title is-4 grid-item-title card-title" id="meep">{{title}}</h3>
```

Comment:
Guessing you don't want to keep `id="meep"` 😄

Reply:
🤦‍♀️. I just need to do a cntrl find for meep everytime I do a PR.  Thanks!